<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Multiplayer</class>
 <widget class="QWidget" name="Multiplayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>多人遊戲大廳</string>
  </property>
  <widget class="QPushButton" name="btn_create">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>120</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>創建房間</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>120</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_join">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>120</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>加入房間</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>260</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>127.000.000.1</string>
   </property>
   <property name="maxLength">
    <number>15</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
